tests/synoptic_tests.py

```diff
@@ -38,14 +38,10 @@ def test_GIVEN_synoptic_THEN_targets_that_it_defines_appear_in_opi_info(self):
             self.fail("In synoptic {}, XML failed to parse properly. Error text was: {}".format(self.synoptic, e))
 
         for type, target in type_target_pairs:
-            if Settings.name == "ZOOM" \
-                    and self.version_utils.get_version() == "6.0.0" \
-                    and target == "LSI correlator":
-                continue  # This is hotfixed on ZOOM. This condition can be removed at next release.
-            if Settings.name == "EMU" \
-                    and (self.version_utils.get_version() == "6.0.0" or self.version_utils.get_version() == "6.0.1") \
-                    and target == "Lakeshore 372":
-                continue  # This is hotfixed on ZOOM. This condition can be removed at next release.
+            if Settings.name == "RIKENFE" \
+                    and self.version_utils.get_version() == "12.0.1" \
+                    and (target == "RIKEN Vacuum " or target == "Riken Kicker and Separator HV settings"):
```

Comment:
is the space after "Vacuum" correct?

Reply:
@davidkeymer can the space be removed on RIKENFE?

Reply:
Have removed space as requested.
Will edit `opi_info.xml` accordingly and add another PR for flash review.

Reply:
PR here: https://github.com/ISISComputingGroup/ibex_gui/pull/1528